README.rst

```diff
@@ -1,4 +1,4 @@
-Girder Worker |build-status| |docs-status| |license-badge| |gitter-badge|
+|logo| Girder Worker |build-status| |docs-status| |license-badge| |gitter-badge|
```

Comment:
It's actually ok as is (see https://github.com/girder/girder_worker/tree/logo). Markdown is pretty flexible about these things. Nonetheless, I'll commit a fix in a moment.